### PR TITLE
Add com.github.johnfactotum.Foliate

### PR DESCRIPTION
**Pull request by johnfactotum**
Foliate is a simple and modern GTK eBook reader.

https://github.com/johnfactotum/foliate

### CODE REVIEW

File: com.github.johnfactotum.Foliate.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [
+        {
+            "name": "foliate",
+            "builddir": true,
```
Comment (bilelmoussaoui) [inline com.github.johnfactotum.Foliate.json]:
```suggestion
```
It's already set by default

**Comment by bilelmoussaoui** [comment]
bot, build com.github.johnfactotum.Foliate

**Comment by bilelmoussaoui** [comment]
The build fails because the appdata is lacking those tags
```
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• tag-missing           : <release> required
```

File: com.github.johnfactotum.Foliate.json (hunk 1 above)
Comment (johnfactotum) [reply com.github.johnfactotum.Foliate.json]: Thanks. Wasn't aware of this. I just sort of blindly copied it from Gnome Builder's template.

**Comment by johnfactotum** [comment]
Well, this is embarrassing. I should've validated it. It should be fixed now.

**Comment by LeandroStanger** [comment]
bot, build com.github.johnfactotum.Foliate

**Comment by nedrichards** [comment]
@johnfactotum there's a spelling mistake on the keyboard shortcuts page. it should say `increase font size` not `fotn` (not a blocker, but you'd probably appreciate knowing)

**Comment by nedrichards** [comment]
> libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open drm device: No such file or directory
libGL error: failed to load driver: i965

This error means you need `--device=dri` to get access to GPU acceleration.

**Comment by nedrichards** [comment]
Other than that, this looks really nice. I look forward to adding it when that nit is fixed.

**Comment by johnfactotum** [comment]
Thanks for the catch. Fix committed.

I've dded `--device=dri` to the manifest. Thank you for the kind words!

**Comment by nedrichards** [comment]
bot, build com.github.johnfactotum.Foliate

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.johnfactotum.Foliate

**Pull request closed without merging**